deploy/operator.yaml

```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: wildfly-operator
       containers:
         - name: wildfly-operator
-          image: quay.io/wildfly/wildfly-operator:latest
+          image: 172.30.1.1:5000/wildfly-operator-testing/wildfly-operator
```

Comment:
I believe you've missed this one.

Reply:
+1, thanks. that was a bit hasty. The change is in my git staging area :man_facepalming: